beta-src/src/state/game/game-api-slice.ts

```diff
@@ -374,23 +422,31 @@ const gameApiSlice = createSlice({
             },
           };
           setCommand(state, command, "mapCommands", "all");
-          if (currentOrders) {
-            const orderToUpdate = currentOrders.find(
-              (o) => o.unitID === currOrderUnitID,
-            );
-            if (orderToUpdate) {
-              updateOrdersMeta(state, {
-                [orderToUpdate.id]: {
-                  saved: false,
-                  update: {
-                    type: "Hold",
-                    toTerrID: null,
+
+          if (currentOrders?.length) {
+            phase === "Retreats"
+              ? updateOrdersMeta(state, {
+                  [order.orderID]: {
+                    saved: false,
+                    update: {
+                      type: "Disband",
+                      toTerrID: null,
+                    },
                   },
-                },
-              });
-            }
+                })
+              : updateOrdersMeta(state, {
+                  [order.orderID]: {
+                    saved: false,
+                    update: {
+                      type: "Hold",
+                      toTerrID: null,
+                    },
+                  },
+                });
```

Comment:
your doing a ternary here but the only difference i see is type. Can we just simplify to the following

you can do `type:  phase === "Retreats" ?  "Disband" : "Hold"`

Reply:
Idk why I missed that lol